src/test/scala/org/hammerlab/guacamole/util/TestUtil.scala

```diff
@@ -231,11 +232,30 @@ object TestUtil extends Matchers {
       Pileup(normalReads, contig, locus, reference.getContig(contig)))
   }
 
-  def loadPileup(sc: SparkContext, filename: String, reference: ReferenceBroadcast, locus: Long = 0, contig: Option[String] = None): Pileup = {
-    val records = TestUtil.loadReads(sc, filename, reference = reference).mappedReads
+  def loadPileup(sc: SparkContext,
+                 filename: String,
+                 reference: ReferenceBroadcast,
+                 locus: Long = 0,
+                 contigOpt: Option[String] = None): Pileup = {
```

Comment:
nit: `Opt` is an abbreviation (I feel kind of awful being this nit picky but abbreviation creep is so easy to happen and we need to be steadfast)